server/plugins/LFUGens.cpp

```diff
@@ -2302,6 +2312,169 @@ void LinExp_Ctor(LinExp* unit) {
     unit->m_rrminuslo = unit->m_rsrcrange * -srclo;
     LinExp_next(unit, 1);
 }
+////////////////////////////////////////////////////////////////////////////////////////////////////////
```

Comment:
This section was shamelessly copied from `LinExp` above, with the calculations changed. I believe I worked out all of the kinks.

Can anyone give me insight into what the `m_` prefix is on some of the variables created in the constructor? I don't know what the `m_` is short for, but I think it's just the initial value of the arguments passed into the UGen.

Reviewers:
Please look out for any cases where I missed changing a `LinExp` to `LinLin`. I missed a couple in my first iteration, but I think I got them all.
I think I got rid of every instance of the variables created in `LinExp_Ctor`. Please verify this.

Reply:
m_ prefixes are members and is needed to keep style consistent across UGens, please